pom.xml

```diff
@@ -260,17 +262,24 @@
           <includeTests>true</includeTests>
           <xmlOutput>${findbugs.xmlOutput}</xmlOutput>
           <failOnError>${findbugs.failOnError}</failOnError>
-          <skip>${findbugs.skip}</skip>
```

Comment:
Any reason the support for the flag is removed altogether?

Reply:
It is not needed as the execution is not bound to any phase. If findbugs is invoked directly from the command line, there is no reason to skip it.

Reply:
Sounds good, so it does not have an automatic default phase?

Reply:
Please see line 268.